packages/inquirer/src/ui/prompt.mts

```diff
@@ -197,7 +197,6 @@ export default class PromptsRunner<A extends Answers> {
   process: Observable<any> = EMPTY;
   private abortController: AbortController = new AbortController();
   private opt: StreamOptions;
-  rl?: InquirerReadline;
```

Comment:
In v9 this is a singleton. In v10 it's undefined for modern and ephemeral for legacy prompts.